### ISSUE TITLE
Do you support NETLOGON protocol or do you have a plan?

### ISSUE THREAD

**Issue by fengyingkong** [fortra/impacket#1662]
MSRPC-NETLOGON

Do you support NETLOGON protocol or do you have a plan?

**Comment by NtAlexio2** [comment]
Yes, there is **MS-NRPC** implementation in `impacket/dcerpc/v5/nrpc.py` path. Please explain in detail if you have specific meaning.

**Comment by anadrianmanrique** [comment]
closing this one. reopen if needed
